ballerina/tests/test.bal

```diff
@@ -154,8 +154,8 @@ function testListTables() returns error? {
 }
 function testPutItem() returns error? {
     ItemCreateInput request = {
-        tableName: mainTable,
-        item: {
+        TableName: mainTable,
+    Item: {
```

Comment:
```suggestion
        TableName: mainTable,
        Item: {
```

Reply:
Fixed in [95b066a](https://github.com/ballerina-platform/module-ballerinax-aws.dynamodb/pull/49/commits/95b066abe7867ea2939f83aa2d4a41bdcfecc1cb)